tests/parser/test_benchmark.py

```diff
@@ -0,0 +1,330 @@
+                    if b then {
+                        a = a + 2;
+                    } else {
+                        a = a + b + s + 5 + a + b+ 3 + 4 +5 +f +6 +d+ s;
+                    }
+                }
+                        } else {
+                            a = a + b + s + 5 + a + b+ 3 + 4 +5 +f +6 +d+ s;
+                        }
+                    }
+                }
+                    } else {
+                        if b then {
+                            a = a + 2;
+                        } else {
+                            a = a + b + s + 5 + a + b+ 3 + 4 +5 +f +6 +d+ s;
+                        }
+                    }
+                } else {
+                    if b then {
+                        a = a + 2;
+                    } else {
+                        a = a + b + s + 5 + a + b+ 3 + 4 +5 +f +6 +d+ s;
+                    }
+                }
+                        }
+                    }
+                }
+
+                if b then {
+                    if b then {
+                while a {
+                    if b then {
+                        a = a + 2;
+                    } else {
+                        if b then {
+                if b then {
+                    if b then {
+                while a {
+                    if b then {
+                        a = a + 2;
+                    } else {
+                        if b then {
+                            a = a + 2;
+                        } else {
+                            a = a + b + s + 5 + a + b+ 3 + 4 +5 +f +6 +d+ s;
+                        }
+                    }
+                }
+                    } else {
+                        if b then {
+                            a = a + 2;
+                        } else {
+                            a = a + b + s + 5 + a + b+ 3 + 4 +5 +f +6 +d+ s;
+                        }
+                    }
+                } else {
+                    if b then {
+                        a = a + 2;
+                    } else {
+                        a = a + b + s + 5 + a + b+ 3 + 4 +5 +f +6 +d+ s;
+                    }
+                }
+                        } else {
+                            a = a + b + s + 5 + a + b+ 3 + 4 +5 +f +6 +d+ s;
+                        }
+                    }
+                }
+                    } else {
+                        if b then {
+                            a = a + 2;
+                        } else {
+                            a = a + b + s + 5 + a + b+ 3 + 4 +5 +f +6 +d+ s;
+                        }
+                    }
+                } else {
+                    if b then {
+                        a = a + 2;
+                    } else {
+                        a = a + b + s + 5 + a + b+ 3 + 4 +5 +f +6 +d+ s;
+                    }
+                }
+                    } else {
+                        if b then {
+                            a = a + 2;
+                        } else {
```

Comment:
Większej funkcji się nie dało zrobić???

Reply:
Dalo. Jest tylko po to by stestowac czas parsowania. PipeTester ucina (wg screenow) przy >10s, ja ucinam przy >0.1s

Reply:
A da się to zrobić by nie zajmowało tak dużo miejsca?